docset/windows/dnsserver/Export-DnsServerZone.md

```diff
@@ -119,7 +119,7 @@ Accept wildcard characters: False
 
 ### -FileName
 Specifies a name for the export file.
-You can include a file path.
+You CANNOT include a file path. the file will be exported to the SYSTEM32 DNS Folder
```

Comment:
Replace with something like:

File is exported to %windir%\System32\dns.

This will make it clear to the reader where to find the exported file.